app/models/team_member.rb

```diff
@@ -1,5 +1,9 @@
 class TeamMember < ApplicationRecord
   belongs_to :project, optional: true
-  validates_presence_of :name
+  validates_presence_of :first_name, :last_name
```

Comment:
(Not everyone has both)

Reply:
Ah yes, good point! Thanks for highlighting that F. We'll sort that out.

Reply:
I did ponder this, then made the decision to have it mirror what was coming down the pipe from 10,000ft (which does require both).

In an ideal world we'd have `given_name`, `family_name`, and both would be optional.

That said, I don't feel particularly strongly either way about keeping or removing this validation.

Reply:
We've decided to drop the requirement for last_name, but have kept first_name to ensure there is something to display.